src/CommerceModel.BetterRetail/packages.config

```
@@ -40,15 +44,23 @@
   <package id="ServiceStack.OrmLite.SqlServer" version="5.9.0" targetFramework="net461" />
   <package id="ServiceStack.Text" version="5.9.0" targetFramework="net461" />
   <package id="System.Buffers" version="4.5.1" targetFramework="net461" />
+  <package id="System.Diagnostics.DiagnosticSource" version="4.6.0" targetFramework="net461" />
+  <package id="System.IO.Hashing" version="6.0.0" targetFramework="net461" />
   <package id="System.Memory" version="4.5.4" targetFramework="net461" />
+  <package id="System.Memory.Data" version="1.0.2" targetFramework="net461" />
   <package id="System.Net.Http" version="4.3.4" targetFramework="net461" />
   <package id="System.Net.Http.WinHttpHandler" version="4.5.0" targetFramework="net461" />
   <package id="System.Numerics.Vectors" version="4.5.0" targetFramework="net461" />
-  <package id="System.Runtime.CompilerServices.Unsafe" version="4.5.3" targetFramework="net461" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="4.7.1" targetFramework="net461" />
+  <package id="System.Runtime.InteropServices.RuntimeInformation" version="4.3.0" targetFramework="net461" />
   <package id="System.Security.Cryptography.Algorithms" version="4.3.0" targetFramework="net461" />
   <package id="System.Security.Cryptography.Encoding" version="4.3.0" targetFramework="net461" />
   <package id="System.Security.Cryptography.Primitives" version="4.3.0" targetFramework="net461" />
   <package id="System.Security.Cryptography.X509Certificates" version="4.3.0" targetFramework="net461" />
   <package id="System.Spatial" version="5.7.0" targetFramework="net461" />
+  <package id="System.Text.Encodings.Web" version="4.7.2" targetFramework="net461" />
+  <package id="System.Text.Json" version="4.7.2" targetFramework="net461" />
+  <package id="System.Threading.Tasks.Extensions" version="4.5.4" targetFramework="net461" />
+  <package id="System.ValueTuple" version="4.5.0" targetFramework="net461" />
   <package id="WindowsAzure.Storage" version="7.2.1" targetFramework="net461" />
```

```suggestion
  <package id="WindowsAzure.Storage" version="7.2.1" targetFramework="net461" />
```

is this still required?